### PR TITLE
Bump swarm-derive version

### LINKED ISSUE #2746: libp2p-swarm-derive is not updated/published in 0.46
https://github.com/libp2p/rust-libp2p/blob/7df6bae5200dcc31bdc9b1a789571aab72c6d9ac/swarm-derive/Cargo.toml#L6

I got a compilation error:

```
error[E0412]: cannot find type `ListenerId` in module `libp2p::core::connection`
  --> client/network/src/behaviour.rs:65:10
   |
65 | #[derive(NetworkBehaviour)]
   |          ^^^^^^^^^^^^^^^^
   |          |
   |          not found in `libp2p::core::connection`
   |          in this derive macro expansion
   |
  ::: /home/kpp/.cargo/registry/src/github.com-1ecc6299db9ec823/libp2p-swarm-derive-0.27.1/src/lib.rs:30:1
   |
30 | pub fn hello_macro_derive(input: TokenStream) -> TokenStream {
   | ------------------------------------------------------------ in this expansion of `#[derive(NetworkBehaviour)]`

For more information about this error, try `rustc --explain E0412`.
```

Can you please publish a hotfix? ^_^

Related to: https://github.com/libp2p/rust-libp2p/pull/2730

cc @mxinden

### PR DESCRIPTION
**Pull request by kpp**
Fix https://github.com/libp2p/rust-libp2p/issues/2746

### CODE REVIEW

File: swarm-derive/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 0.28.0
+
+- Remove ListenersStream. See [PR 2652].
```
Comment (elenaf9) [inline swarm-derive/CHANGELOG.md]:
```suggestion
- Import `ListenerId` from `libp2p::core::transport`. See [PR 2652].
```

**Review by elenaf9 — APPROVED** [review]
Oh my bad, I missed that version bump in #2652. Thanks @kpp!

**Review by mxinden — APPROVED** [review]
Thanks @kpp for providing the fix.

Missed it in the reviews as well. Sorry about that. I wonder how we can prevent this in the future. This has happened before.

**Comment by mxinden** [comment]
Tagged and published. Thanks for the help!

**Pull request merged**